appscale/tools/remote_helper.py

```diff
@@ -151,31 +153,68 @@ def start_all_nodes(cls, options, node_layout):
 
     agent.configure_instance_security(params)
 
-    load_balancer_nodes = node_layout.get_nodes('load_balancer', True)
-    instance_ids, public_ips, private_ips = cls.spawn_load_balancers_in_cloud(
-      options, agent, params,
-      len(load_balancer_nodes))
+    load_balancer_roles = {}
 
-    for node_index, node in enumerate(load_balancer_nodes):
-      index = node_layout.nodes.index(node)
-      node_layout.nodes[index].public_ip = public_ips[node_index]
-      node_layout.nodes[index].private_ip = private_ips[node_index]
-      node_layout.nodes[index].instance_id = instance_ids[node_index]
+    instance_type_roles = {'with_disks':{}, 'without_disks': {}}
```

Comment:
I'm having trouble figuring out the reason for this separation.

Reply:
@cdonati azure cannot attach existing disks to scaleset machines so they would have to be created as regular machines, it was added in a little early with the expectation of managed disks.

Reply:
Okay. That sounds like an implementation detail that belongs in the azure agent instead of here.